RecoTracker/FinalTrackSelectors/python/MergeTrackCollections_cff.py

```diff
@@ -44,12 +44,13 @@
 generalTracks.candidateComponents = cms.InputTag("duplicateTrackCandidates","candidateMap")
 
 
-generalTracksSequence = cms.Sequence(
-    duplicateTrackCandidates*
-    mergedDuplicateTracks*
-    duplicateTrackClassifier*
+generalTracksTask = cms.Task(
+    duplicateTrackCandidates,
+    mergedDuplicateTracks,
+    duplicateTrackClassifier,
     generalTracks
     )
+generalTracksSequence = cms.Sequence(generalTracksTask)
```

Comment:
Let me ask a conceptual question: why do we need both `Task` and `Sequence` for the cases where the `Sequence` contains only a single `Task`? I.e. what prevents `generalTracksSequence = cms.Task(...)`? (ok, eventually should be named as `generalTracksTask`) Is this just an intermediate step for a full `Task` migration or is there something more fundamental?

Reply:
Correct. In this case it can just be a sequence.

Reply:
It is an intermediate step. If we converted `generalTracksSequence` to be a `Task` directly, then any code that did `+generalTracksSequence+` would break. This way of moving forward keeps all existing configurations working.

Reply:
Ok, thanks.